### PR TITLE
directory lookup should have X-headers

### PR DESCRIPTION
**Pull request by lazedo**

### CODE REVIEW

**Comment by lazedo** [comment]
master https://github.com/2600hz/kazoo/pull/3683

File: applications/ecallmgr/src/ecallmgr_fs_authn.erl [hunk 1]
```diff
@@ -178,23 +178,15 @@ code_change(_OldVsn, State, _Extra) ->
...
-        _Other ->
-            {'ok', Resp} = ecallmgr_fs_xml:empty_response(),
-            _ = freeswitch:fetch_reply(Node, Id, 'directory', Resp),
-            lager:debug("ignoring authn request from ~s for ~p", [Node, _Other])
+    lager:debug("received fetch request (~s) user creds from ~s", [Id, Node]),
+    case kzd_freeswitch:authorizing_id(Props) /= 'undefined'
+        andalso kzd_freeswitch:authorizing_type(Props) /= 'undefined'
+    of
+        'true' ->
+            lookup_user(Node, Id, <<"password">>, Props);
```
Comment (k-anderson) [inline applications/ecallmgr/src/ecallmgr_fs_authn.erl]: @lazedo howdy, what happend to the reverse-lookup?

Comment (lazedo) [inline applications/ecallmgr/src/ecallmgr_fs_authn.erl]: @k-anderson i don't think its asked anymore.

Comment (lazedo) [inline applications/ecallmgr/src/ecallmgr_fs_authn.erl]: @k-anderson reviewing again, don't merge

**Comment by lazedo** [comment]
@k-anderson updated

**Pull request merged**